extensions/interactions/NumericExpressionInput/directives/numeric-expression-input-interaction.component.html

```diff
@@ -3,3 +3,16 @@
 <div ng-show="!$ctrl.isCurrentAnswerValid()" class="alert alert-danger mt-2">
   <[$ctrl.warningText]>
 </div>
+<!-- Using CSS selectors to fix issue -->
+<style>
+  .guppy-div-learner > span > span:nth-child(2) > span {
+    white-space: pre-wrap;
+    width: 100%;
+  }
+
+  .guppy-div-learner > span > span:nth-child(2) > span > span:nth-child(2) {
+    white-space: pre-wrap;
+    word-break: break-all;
+    word-wrap: break-word;
+  }
```

Comment:
Could you please add a more descriptive comment here? Explain why are the selectors so complicated and what this code does in the end. "Using CSS selectors to fix issue" is not really helpful since the reader doesn'ŧ know what is/was the issue.

Also would it be possible to use `hyphens` instead of/together with `word-break` so that the words are properly hyphenated?

Reply:
I have added better comments to describe the exact issue being faced and how I am using a solution. It should be more readable now.

Reply:
Also now I'm using hyphens instead of word-break. The effect seems to be better.